README.md

```diff
@@ -8,20 +8,55 @@ Link Picker control designed to work with CMB2.
 
 Using the Link Picker for [CMB2](https://wordpress.org/plugins/cmb2/) control, you can choose a link from your WordPress site, or manually enter a link. You can also identify if the link should open in a new window, or not.
 
-Features:
+**Features:**
 
 * Easy to integrate with [CMB2](https://wordpress.org/plugins/cmb2/), just add a type of `link_picker`
 * Works with repeatable groups
 * Works as a repeatable field when `repeatable` is set to `true`
 * Outputs an array of `text`, `url` and `blank` when using `get_post_meta`
 * You are able to split the values of the field into individual parts by setting `split_values` to `true`. You can retrieve the split values by using the ID of the field and appending `_text`, `_url` and `_blank` to the ID when using `get_post_meta` (not compatible if using a repeatable field)
 
+See usage examples under Installation.
+
 ## Installation
 
 1. Download this repository and unzip it into the folder `link-picker-for-cmb2`
 2. Upload the `link-picker-for-cmb2` folder to the `/wp-content/plugins/` directory
 3. Activate the plugin through the 'Plugins' menu in WordPress
-4. Configure the plugin via the 'Link Picker for CMB2' options page under the WordPress 'Settings' Menu
+4. There are no settings for the plugin.  See examples below for usage.
+
+Example metabox:
+
+```
+$cmb->add_field( array(
+	'name' => __( 'Website URL', 'cmb2' ),
+	'desc' => __( 'field description (optional)', 'cmb2' ),
+	'id'   => $prefix . 'url',
+	'type' => 'link_picker',
+	'repeatable' => true,
+	'split_values' => true  // default is false
+) );
+```
+
+Example display:
+
+```
+$url = get_post_meta( $post_ID, '_yourprefix_url', true );
+
+
+if ( 'true' == $url[0]['blank'] ) {
+	$blank = ' target="_blank"';
+} else {
+	$blank = '';
+}
+
+printf(
```

Comment:
For clarity of code, I always prefer to use %1s, %2s etc.. in a printf, however most of the time I will break out of the php and use:

```
?>
<a href=""...
<?php
```

I just find this easier to read. But that is my preference and not wrong at all.